ci-operator/config/openstack-k8s-operators/openstack-k8s-operators-ci/openstack-k8s-operators-openstack-k8s-operators-ci-main.yaml

```diff
@@ -49,6 +57,24 @@ images:
       as:
       - registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.18-openshift-4.12
   to: ci-build-root-golang-1.19-sdk-1.31
+- build_args:
+  - name: SDK_VERSION
+    value: v1.31.0
+  dockerfile_path: images/build-root/Dockerfile.custom
+  inputs:
+    rhel-8-golang-1.20:
```

Comment:
this now needs to match the definition added in `base_images` - so it needs to be `rhel-9-golang-1.20`